### PR TITLE
1454 json schema

### PR DESCRIPTION
**Pull request by mrix**
* Updated JSON schema
* Added `Graviton\JsonSchemaBundle`
* Updated bin files

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,21 +13,26 @@
...
       "./vendor/bin/phpcs --config-set installed_paths ../../libgraviton/codesniffer/CodeSniffer/Standards",
       "./vendor/bin/phpcs --standard=PSR1 src/",
       "./vendor/bin/phpcs --standard=PSR2 src/",
       "./vendor/bin/phpcs --standard=ENTB src/"
     ]
   },
+  "bin": [
+    "bin/graviton-validate-directory",
+    "bin/graviton-validate-file"
+  ],
```
Comment (hairmare) [inline composer.json]: It seems ``bin/graviton-validate-schema`` is missing...

Comment (mrix) [inline composer.json]: `bin/graviton-validate-schema` validates our JSON schema.
But `justinrainbow/json-schema` library has a bug in `RefResolver`. I opened [new issue](https://github.com/justinrainbow/json-schema/issues/185) about it.

**Comment by narcoticfresh** [comment]
@mrix yes, `justinrainbow/json-schema` has some bugs.. couldn't we use the same one here you use in graviton?

**Comment by mrix** [comment]
@narcoticfresh In graviton we use https://github.com/HadesArchitect/JsonSchemaBundle that depends on the `justinrainbow/json-schema` library

**Comment by narcoticfresh** [comment]
@mrix
* ACK to your point regarding the bundles..
* The missing script that @hairmare refers to is `bin/graviton-validate-schema` - it is missing in composer.json, is this on purpose?
* Regarding the issue you did in `justinrainbow/json-schema`: If you check the reaction time there on issues, I don't expect a fast reply. Could you do a PR for the bug there that affects us? And please, don't also refactor everything there :grin:

**Comment by mrix** [comment]
I created new PR https://github.com/justinrainbow/json-schema/pull/186

**Comment by lapistano** [comment]
regarding the PR: if they do not react 'fast' there is always the possibility to fork both repositories (`hadesarchitect/json-schema-bundle` and `justinrainbow/json-schema`) and use composer aliases to route to our forks. I now it is not the nicest way, but it is an easy way to get the patches working. WDYT?

**Comment by narcoticfresh** [comment]
@lapistano 
yes, that's correct. currently, we have no urgency for this PR - we can wait a bit for `justinrainbow/json-schema`

**Comment by narcoticfresh** [comment]
@mrix 
I think we can merge this PR already as it's a major improvement over the current state of `develop` in this repo.. could you please first create an issue in this repo (`libgraviton/json-schema`), shortly documenting what we're waiting for in `justinrainbow/json-schema` and what we next to fix next?

After that, I'll merge this and we can resume this. Eventually, I'll make a 0.1 tag so we can incorporate it in graviton already (the schema)..

**Comment by hairmare** [comment]
I just realized that ``bin/graviton-validate-schema`` is being executed during composer check, now I get it.

**Pull request merged**